### DISCUSSION TITLE
How to get Product Pricing info

### DISCUSSION THREAD

**Discussion by LouisPyCh** [saleweaver/python-amazon-sp-api#928] [Q&A]
Hey,
I have tried to get information on product pricing with the following code:

from sp_api.api import Products
from sp_api.base import Marketplaces

CLIENT_CONFIG =dict(
refresh_token='Atzr|...',
aws_access_key_id='...',
aws_secret_access_key='...',
lwa_app_id='...',
lwa_client_secret='...',
role_arn='...'
)

res_product = Products(credentials=CLIENT_CONFIG, marketplace=Marketplaces.DE)
res = res_product.get_product_pricing_for_skus(['SKU'], MarketplaceId="A1PA6795UKMFR9")

print(res)
print(res.payload)

However I get an error saying: "ImportError: cannot import name 'Products' from partially initialized module 'sp_api.api' (most likely due to a circular import)"

Does anybody know what I am doing wrong? I use PyCharm and am very new to Python.

Thanks a lot! :)

**Comment by tilschuenemann** [comment]
@LouisPyCh Is the file you are running named "products.py" or is there another file named like that in the current working directory? Please let me know.
